plugins/commander.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: commander
+spec:
+  homepage: https://github.com/schabrolles/kubectl-commander.git
```

nit: 
```suggestion
  homepage: https://github.com/schabrolles/kubectl-commander
```

if you fix this, https://krew.sigs.k8s.io/plugins/ can show a repository badge for you.